packages/gateway/src/config/routes.ts

```diff
@@ -127,82 +55,15 @@ export const getRoutes = () => {
         }
       }
     },
-    // Authentication routes. These are proxied to the auth service
-    {
-      method: 'POST',
-      path: '/auth/{suffix}',
-      handler: (_, h) =>
-        h.proxy({
-          uri: AUTH_URL + '/{suffix}'
-        }),
-      options: {
-        auth: false,
-        payload: {
-          output: 'data',
-          parse: false
-        }
-      }
-    },
-    {
-      method: 'POST',
-      path: '/auth/authenticate',
-      handler: rateLimitedRoute(
-        { requestsPerMinute: 10, pathForKey: 'username' },
-        (_, h) =>
-          h.proxy({
-            uri: AUTH_URL + '/authenticate'
-          })
-      ),
-      options: {
-        auth: false,
-        payload: {
-          output: 'data',
-          parse: false
-        }
-      }
-    },
-    {
-      method: 'POST',
-      path: '/auth/authenticate-super-user',
-      handler: rateLimitedRoute(
-        { requestsPerMinute: 10, pathForKey: 'username' },
-        (_, h) =>
-          h.proxy({
-            uri: AUTH_URL + '/authenticate-super-user'
-          })
-      ),
-      options: {
-        auth: false,
-        payload: {
-          output: 'data',
-          parse: false
-        }
-      }
-    },
-    {
-      method: 'POST',
-      path: '/auth/verifyUser',
-      handler: rateLimitedRoute(
-        { requestsPerMinute: 10, pathOptionsForKey: ['mobile', 'email'] },
-        (_, h) =>
-          h.proxy({
-            uri: AUTH_URL + '/verifyUser'
-          })
-      ),
-      options: {
-        auth: false,
-        payload: {
-          output: 'data',
-          parse: false
-        }
-      }
-    }
+
+    catchAllProxy.locations,
+    catchAllProxy.locationId,
+
+    catchAllProxy.auth,
+    rateLimitedAuthProxy.authenticate,
+    rateLimitedAuthProxy.authenticateSuperUser,
+    rateLimitedAuthProxy.verifyUser
   ]
-  // add all routes from all modules to the routes array manually or write your routes inside a folder inside the server folder
-  // with suffix as -routes.ts
-  glob.sync(join(__dirname, '../routes/**/*-route.[t|j]s')).forEach((file) => {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    routes.push(require(resolve(file)).default)
-  })
```

Comment:
We don't use the file-based routes so I removed the possibility. Let's rather import the routes explicitly for clarity

Reply:
Good